.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Do you need this now? It apparently causes a conflict. I think you can add it later, given that #107 contains
```yaml
  document-start: disable
```

Reply:
I will resolve the conflict. It was already part of #106